Exercise-HTML-CSS-1/RicardoGlz/Excersise1.html

```diff
@@ -1,133 +1,142 @@
-                            </li>
-                            <li>
-                                <a href="#">Austin</a>
-                            </li>
-                            <li>
-                                <a href="#">Philly</a>
-                            </li>
-                        </ul>
-                    </li>
-                    <li>
-                        <a>World</a>
-                        <ul>
-                            <li>
-                                <a href="#">Otawwa</a>
-                            </li>
-                            <li>
-                                <a href="#">Sydney</a>
-                            </li>
-                        </ul>
-                    </li>
-                </ul>
-            </div>
-            <div class="Upcoming-Classes">
-                <h1>Upcoming Classes</h1>
-                <table border="1" class="Tabla">
-                    <thead>
-                        <tr>
-                            <th>Location</th>
-                            <th>Topic</th>
-                        </tr>
-                    </thead>
-                    <tbody>
-                        <tr>
-                            <td>New York</td>
-                            <td>JavaScript and Jquery</td>
-                        </tr>
-                        <tr>
-                            <td>San Francisco</td>
-                            <td>HTML and CSS</td>
-                        </tr>
-                    </tbody>
-                </table>
-            </div>
-            <div class="Sign-Up">
-                <h1>Sign Up!</h1>
-                <form>
-                    <label> 
-                        Name: <input type="text" name="Name" id="Name"> 
-                    </label>
-                </form>
-                <form>
-                    <label> 
-                        Email: <input type="text" name="Email" id="Email"> 
+        <div class="Container">
+                <div class="Wrapper">
+                <div class="Title">
+                    <h1>About</h1>
+                </div>
+                <div class="Information">
+                    <p>
+                        Lorem ipsum dolor sit amet, consectetur adipiscing 
+                        elit. Suspendisse ut lorem at ex porttitor lacinia 
+                        in non diam. Phasellus eros lectus, tristique vitae 
+                        pellentesque ut, rhoncus vel velit. Donec rutrum 
+                        ante ac velit porttitor placerat non non tortor. 
+                        Sed tempor risus in quam blandit pharetra sit amet 
+                        eget libero. Nunc ullamcorper, augue ac auctor
+                        facilisis, augue metus ultricies ipsum, quis 
+                        ullamcorper lorem lectus id velit. Nulla eu enim purus.
+                    </p>
+                </div>
+                <div class="Title-Location">
```

Comment:
Please use only lower case to name clases (Ex. `title-location`). Also try to always use English to name classes and id's